src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
-    url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
-    description={`Read about Material Design's navigation fundamentals.`}
-/>
-<MaterialDesignDescription
-    title={'Navigation Drawer '}
-    url={'https://material.io/components/navigation-drawer/'}
-    style={{marginRight: 0}}
+Navigation refers to the way in which users access the different areas of your application and move from screen to screen. The most common forms of primary navigation are drawers and tabs.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Understanding Navigation'}
+        url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
+        description={`Read about Material Design's navigation fundamentals.`}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached on the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid images={[NavigationBreakdownImg]} regularWidth />
+
+Most drawers have four main sections:
+
+-   **Header**: contains the name of the company, name of the product or a logo
+-   **Subheader**: optional section that can be used to support custom content, such as filtering or search
+-   **Body**: contains a list of primary pages that users can navigate too
+-   **Footer**: contains company logo, version number and trademark information
+
+There are several different styles of drawer navigation, each with slightly different behavior.
+
+<DemoCard
+    repository={'Drawer'}
+    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
 />
 
-## Fixed Navigation
+## Permanent
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
 
-## Collapsible Navigation
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
 
-A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
+<ImageGrid images={[NavigationPermanentImg]} regularWidth />
```

Comment:
![image](https://user-images.githubusercontent.com/8997218/94961549-d9d95e80-04c2-11eb-98f2-87d0ab824ac6.png)

This shadow is blurred... use the latest shadow in DSM

Reply:
updarted